toolkits/global/packages/global-corporate-footer/scss/10-settings/default.scss

```diff
@@ -4,12 +4,20 @@
  *
  */
 
+$corporate-footer--vertical-rhythm: spacing(32);
+
+$corporate-footer--margin-bottom: spacing(0);
+$corporate-footer--border: 2px solid #dadada;
+$corporate-footer--padding-top: $corporate-footer--vertical-rhythm;
+$corporate-footer--padding-bottom: $corporate-footer--vertical-rhythm;
 $corporate-footer--font-family: $context--font-family-sans;
-$corporate-footer--background-color: white;
-$corporate-footer--border: 1px solid lightgrey;
-$corporate-footer--legal-padding-top: spacing(4);
+$corporate-footer--background-color: #01324b;
```

Comment:
Is this colour (and the one at line 10) unique to the footer? or are they colours that we find in any of the `brand-context` includes

Reply:
That is the tricky part. I believe that brand colors live in their respective contexts.
Although we want to enforce this branding across brands, the color I think exists in SN context

Reply:
Let's leave this as it is for now and I'll create an issue to look later as think what needs to happen is the [corporate colours](https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/springernature/scss/10-settings/colors/default.scss#L31) from the SN brand need to move to the [default context](https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/default/scss/10-settings/colors/shared.scss)